src/geometry.cpp

```diff
@@ -3,11 +3,14 @@
 // C++ Standard Library
 #include <cmath>
 #include <vector>
+#include <stdexcept>
```

Comment:
Are you throwing exceptions?

Reply:
No, I was thinking about something, but I didn't do it.